tapscript/send.go

```diff
@@ -105,10 +106,13 @@ var (
 // createDummyOutput creates a new Bitcoin transaction output that is later
 // used to embed a Taproot Asset commitment.
 func createDummyOutput() *wire.TxOut {
-	// The dummy PkScript is the same size as an encoded P2TR output.
+	// The dummy PkScript is the same size as an encoded P2TR output and has
+	// a valid P2TR prefix.
 	newOutput := wire.TxOut{
-		Value:    int64(DummyAmtSats),
-		PkScript: make([]byte, 34),
+		Value: int64(DummyAmtSats),
+		PkScript: append(
+			[]byte{0x51, 0x20}, bytes.Repeat([]byte{0x00}, 32)...,
```

Comment:
Maybe move `GenesisDummyScript` into this package so it can be reused?

Reply:
Will just move to `asset`.